machinist/rpc/machinist.proto

```diff
@@ -54,6 +54,23 @@ message DownloadResponse {
   bytes data = 1;
 }
 
+message ControlPlaneStateRequest {
+
+}
+
+// StaticMachine is a singular machine usually pushed from the controlplane. It represents an immutable object
+// scraped from the network
+message ControlPlaneStaticMachine {
```

Comment:
Should this be named just `StaticMachine` - is the `ControlPlane` prefix needed to disambiguate it from another kind of static machine?